src/storage/conversion/ConstantConverter.ts

```diff
@@ -19,26 +38,41 @@ import type { RepresentationConverterArgs } from './RepresentationConverter';
 export class ConstantConverter extends RepresentationConverter {
   private readonly filePath: string;
   private readonly contentType: string;
+  private readonly options: ConstantConverterOptions;
 
   /**
    * Creates a new constant converter.
    *
    * @param filePath - The path to the constant representation.
    * @param contentType - The content type of the constant representation.
+   * @param options - Extra options for the converter.
    */
-  public constructor(filePath: string, contentType: string) {
+  public constructor(filePath: string, contentType: string, options: ConstantConverterOptions) {
     super();
     this.filePath = filePath;
     this.contentType = contentType;
+    this.options = { ...options };
   }
 
-  public async canHandle({ preferences, representation }: RepresentationConverterArgs): Promise<void> {
+  public async canHandle({ identifier, preferences, representation }: RepresentationConverterArgs): Promise<void> {
     // Do not replace the representation if there is no preference for our content type
     if (!preferences.type) {
       throw new NotImplementedHttpError('No content type preferences specified');
     }
-    if (!matchesMediaPreferences(this.contentType, { ...preferences.type, '*/*': 0 })) {
+
+    // Only convert when the options allow it
+    if (isContainerIdentifier(identifier) && !this.options.container) {
```

Comment:
Store `const isContainer`?

Reply:
Do you mean storing `isContainer` as a class parameter instead of `options` (which would require us to have 3 new variables instead of 1), or having `const isContainer = isContainerIdentifier(identifier) && !this.options.container;` and then `if (isContainer)`?

Reply:
I meant `const isContainer = isContainerIdentifier(identifier)` and then reuse that for both `if`s, so we don't have to call twice.